validation/data/gmat_astrodynamics/gmat_scripts/scenario001-mission-sequence.script

```diff
@@ -0,0 +1,141 @@
+% Copyright Loft Orbital Solutions Inc.
```

Comment:
Storage of these files here is temporary (except for testing of the app) agreed?

Reply:
You mean the gmat script files? I am keeping them for now because they are handy to use when debugging is needed but yes eventually there's no need for them.

Reply:
> but yes eventually there's no need for them.

✅